frontend/lib/src/controllers/admin_user_list_controller.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:frontend/src/models/user.dart';
+import 'package:frontend/src/services/user_service.dart';
+import 'package:get/get.dart';
+
+class AdminUserListController extends GetxController {
+  RxList<User> users = <User>[].obs;
+
+  @override
+  void onInit() async {
+    super.onInit();
+    await getUsers();
```

Comment:
Is it possible to just call `users.assignAll(await UserService.getUsers())` inside `onInit()` and remove the `getUsers()` method?